### PR TITLE
[UI] Icon change for clearing the notification 

### LINKED ISSUE #6560: Change the icon for clearing the notification.
#### Current Behavior
Change the icon for clearing the notification.
Screenshot 2022-11-22 193914

#### Desired Behavior
Screenshot 2022-11-22 194322

#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by suhail34**
Signed-off-by: suhail34 <suhailkna@gmail.com>

**Notes for Reviewers**

This PR fixes #6560 

Changed the icon for clearing the notification from tick mark to this :
image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thanks, @suhail34!

**Review by leecalcote — CHANGES_REQUESTED** [review]
Oh! @suhail34, it should be mentioned that we've recently steered Meshery UI in the direction away from use of FontAwesome and MUI icon libraries. Instead, to ensure all icons scale on zoom in/zoom out, Meshery UI is transitioning to only use SVGs directly. Please see this PR as an example - https://github.com/meshery/meshery/pull/6578. In light of this, will you you refactor this PR?

**Review by leecalcote — APPROVED** [review]

**Review by Harshit-Dandriyal — CHANGES_REQUESTED** [review]
@suhail34 I appreciate your efforts, but I think you missed here a little :pinching_hand: 

https://user-images.githubusercontent.com/56961488/205397318-e287c0c6-5427-44b8-b3a2-83ca0c6e33a7.mp4

File: ui/components/MesheryNotification.js [hunk 1]
```diff
@@ -343,8 +343,15 @@ class MesheryNotification extends React.Component {
                         color="inherit"
                         className={classes.drawerButton}
                         onClick={this.handleClearAllNotifications()}
+                        onMouseOver={(e) => {
+                          e.target.style.fill = "black"
+                          console.log(e.target.style.fill)
```
Comment (asubedy) [inline ui/components/MesheryNotification.js:348]:
```suggestion

```

**Review by asubedy — CHANGES_REQUESTED** [review]
Missed to remove console log here

**Comment by leecalcote** [comment]
Suggestion merged, @asubedy 👍

**Comment by asubedy** [comment]
Thank you @leecalcote, the changes looks good here to merge. The video posted by @Harshit-Dandriyal, is not visible though.

**Comment by Harshit-Dandriyal** [comment]
> Thank you @leecalcote, the changes looks good here to merge. The video posted by @Harshit-Dandriyal, is not visible though.
> 

I think safari does not support mp4, because it is showing in my chrome. Let me do something here.

**Comment by Harshit-Dandriyal** [comment]
> I've previously informed @Harshit-Dandriyal that his video recordings are not visible in Safari. Hopefully, @Harshit-Dandriyal will change encoding or change recorders so that others can freely collaborate with him. Otherwise, I won't be watching his videos either, I'm afraid.

I am starting to find other recorder then. 👍

**Comment by suhail34** [comment]
Thanks for pointing out I didn't noticed that Is this Ok

https://user-images.githubusercontent.com/77241166/205497082-315b6626-c44d-49b7-9f7b-1c59a2a8aa1f.mp4

**Comment by Harshit-Dandriyal** [comment]
LGTM, Nice work @suhail34

**Review by Harshit-Dandriyal — APPROVED** [review]

**Review by asubedy — APPROVED** [review]
LGTM, too 👍

**Review by Chadha93 — DISMISSED** [review]
LGTM, Let's get you covered on DCO. :wink:

**Comment by Chadha93** [comment]
Merge conflicts.... @suhail34

**Comment by suhail34** [comment]
> LGTM, Let's get you covered on DCO. wink

will this command get my all commits signed off
`git rebase HEAD~8 --signoff`

**Comment by Chadha93** [comment]
> > LGTM, Let's get you covered on DCO. wink
> 
> will this command get my all commits signed off `git rebase HEAD~8 --signoff`

Yes, this should.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@suhail34 The `package` changes and `go.sum` are not required. You can revert those, then we are good to go with this PR.

**Comment by suhail34** [comment]
I didn't made any changes in those file How do i revert

**Comment by Chadha93** [comment]
> I didn't made any changes in those file How do i revert

@sahilsuman933 You can compare the changes with the current file and revert or simply create a new PR. Don't worry about the DCO. We will get you covered on it.

**Review by Chadha93 — APPROVED** [review]
LGTM. Thank you @suhail34

**Comment by Chadha93** [comment]
Yay @suhail34 :balloon:

**Comment by suhail34** [comment]
Thanks for guiding learned a lot

**Pull request merged**